ngmix/priors.py

```diff
@@ -2097,26 +2104,31 @@ def get_fdiff_scalar(self, val):
         """
         get the probability of the point
         """
+
+        p=self.get_lnprob_scalar(val)
+
+        p = -2*p
+        if p < 0.0:
+            p = 0.0
+        return sqrt(p)
+
+
         """
         from math import erf
 
         p1 = erf((self.maxval-val)/self.width_at_max)
         p1 -= 1.0
-        p1 *= -BIGVAL/2
+        #p1 *= -BIGVAL/2
+        p1 *= -1
 
         p2 = erf((val-self.minval)/self.width_at_min)
         p2 -= 1
-        p2 *= -1
-        p2 *= -BIGVAL/2
+        #p2 *= -1
+        #p2 *= -BIGVAL/2
 
         return p1+p2
         """
```

Comment:
What is this block of commented code doing here?

Reply:
I'm still on the fence how best to implement fdiff for this kind of prior